backend/clutch-config.yaml

```diff
@@ -35,6 +35,19 @@ modules:
   - name: clutch.module.k8s
   - name: clutch.module.kinesis
 services:
+  - name: clutch.service.auditsink.slack
```

Comment:
will remove these changes before merging. showing an example of how the set up will look like in the `clutch-config`